Big-Spender/readme.md

```diff
@@ -109,10 +126,12 @@ INSERT YOUR QUERY HERE
 
 **Claire:** But I think I _only_ want those two dates, not a range, or all the days in between.
 
-**You:** Then you need an extra clause. Here's the query:
+**You:** Then you need an extra c  lause. Here's the query:
 
 ```sql
 CREATE YOUR QUERY HERE
+
+select sum(amount), date from                                                                                                                            from spends                                                                                                                                        where date = '2021-03-01' or date = '2021-04-01'                                                                                                                   group by date;
 ```
```

Comment:
Nice job with the date. How else could we do this? could we use IN?

Reply:
SELECT SUM(amount), date
FROM spends
WHERE date IN ('2021-03-01', '2021-04-01')
GROUP BY date;